Controllers/HomeController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Zoltu.Bags.Client.Controllers
+{
+	public class HomeController : Controller
+	{
+		[Route("", Name ="Home", Order = 0)]
+		[Route("app", Name = "app", Order = 0)]

```

Comment:
Name and Order in the route are parameters of Routing attribute. Name is used to name the route to identify each route uniquely. Order is given to define precedence of routes over other.
Since there is no any other routing, we can remove these.

Removed both attributes.